src/compiler/scala/tools/nsc/transform/Delambdafy.scala

```diff
@@ -76,36 +75,25 @@ abstract class Delambdafy extends Transform with TypingTransformers with ast.Tre
       referrers
     }
 
-    val accessorMethods = mutable.ArrayBuffer[Tree]()
-
-    // the result of the transformFunction method. A class definition for the lambda, an expression
-    // insantiating the lambda class, and an accessor method for the lambda class to be able to
-    // call the implementation
-    case class TransformedFunction(lambdaClassDef: ClassDef, newExpr: Tree, accessorMethod: Tree)
+    // the result of the transformFunction method.
+    sealed abstract class TransformedFunction
```

Comment:
Ah, found it here: https://github.com/scala/scala/compare/1aa97cb...retronym:topic/indylambda-emit-indy#diff-5db7b371d6f56477262acc99308906fdR82